docs/Apps/Metabolomic Data/Dual Mode Visualization.md

```diff
@@ -56,29 +56,9 @@ To process dual mode data, the following files are required:
 *   An already processed .gct file can also serve as input to the app.
 *   The internal standard file is optional.
 
-##Steps in data processing
-
-*   Perform peak annotation on El-MAVEN
-*   Prepare the cohort mapping file
-*   Perform normalization and scaling on the data
-*   View the quality checks for internal standards, metabolites and samples
-*   Perform differential expression between two cohort conditions
-*   Generate input for [IntOmix](https://docs.elucidata.io/Apps/Multi-omic%20Data/IntOmix.html) to identify significantly changing metabolic network modules between two experimental conditions
-
-##Caveats
```

Comment:
Put this as a Caveat
Pathways analysis only works when the data has KEGG Ids within the “compoundId” column.